### PR TITLE
feat: command bar results key down should exit/refocus input

### PR DESCRIPTION
**Pull request by daveleek**
Refocuses command bar input field if command bar results has focus and user starts to type. Closes if Escape button clicked

### CODE REVIEW

File: frontend/src/component/commandBar/CommandBar.tsx [hunk 1]
```diff
@@ -194,13 +194,18 @@ export const CommandBar = () => {
...
     useKeyboardShortcut({ key: 'Escape' }, () => {
-        if (searchContainerRef.current?.contains(document.activeElement)) {
-            searchInputRef.current?.blur();
-        }
+        setShowSuggestions(false);
     });
     const placeholder = `Command bar (${hotkey})`;
 
     useOnClickOutside([searchContainerRef], hideSuggestions);
+    const onKeyDown = (event: any) => {
```
Comment (daveleek) [inline frontend/src/component/commandBar/CommandBar.tsx:202]: Couldn't find a good event type for this that let me use event.key and event.keyCode, KeyboardEvent would but isn't compatible with the onKeyDown prop

File: frontend/src/component/commandBar/CommandBar.tsx [hunk 2]
```diff
@@ -194,13 +194,18 @@ export const CommandBar = () => {
...
+        if (event.key === 'Escape') {
+            setShowSuggestions(false);
+        } else if (event.keyCode >= 48 && event.keyCode <= 110) {
```
Comment (daveleek) [inline frontend/src/component/commandBar/CommandBar.tsx:205]: Anything outside these we should be interested in accepting?

Comment (sjaanus) [reply frontend/src/component/commandBar/CommandBar.tsx:205]: Maybe this event: React.KeyboardEvent<HTMLSpanElement>

**Review by sjaanus — APPROVED** [review]

File: frontend/src/component/commandBar/CommandBar.tsx (hunk 2 above)
Comment (daveleek) [reply frontend/src/component/commandBar/CommandBar.tsx:205]: Cool thanks, I wasn't aware there was a difference between React.KeyboardEvent and KeyboardEvent. That works! I'll push that improvement and merge

**Pull request merged**